pkg/local_object_storage/metabase/select.go

```diff
@@ -271,6 +307,56 @@ func (db *DB) selectFromFKBT(
 	}
 }
 
+// selectOutsideFKBT looks into all incl buckets to find list of addresses outside <fkbt> to add in
+// resulting cache.
+func selectOutsideFKBT(
+	tx *bbolt.Tx,
+	incl [][]byte, // buckets
+	name []byte, // fkbt root bucket name
+	f object.SearchFilter, // filter for operation and value
+	prefix string, // prefix to create addr from oid in index
+	to map[string]int, // resulting cache
+	fNum int, // index of filter
+) {
+	exclBktRoot := tx.Bucket(name)
+
+	mExcl := make(map[string]struct{})
+
+	bktExcl := tx.Bucket(name)
+	if bktExcl != nil {
+		_ = bktExcl.ForEach(func(k, _ []byte) error {
+			exclBktLeaf := exclBktRoot.Bucket(k)
```

Comment:
Can't you do `exclBktLeaf := bktExcl.Bucket(k)` there?

Reply:
Yes, fixed